src/lib/y2partitioner/widgets/lvm_pv_table.rb

```diff
@@ -0,0 +1,74 @@
+require "yast"
+
+require "cwm/table"
+
+require "y2partitioner/icons"
+require "y2partitioner/widgets/blk_devices_table"
+
+module Y2Partitioner
+  module Widgets
+    # Table widget to represent given list of Y2Storage::LvmLvs together.
+    class LvmPvTable < CWM::Table
+      include BlkDevicesTable
+
+      # @param pvs [Array<Y2Storage::LvmPv] devices to display
+      # @param pager [CWM::Pager] table have feature, that double click change content of pager
+      #   if someone do not need this feature, make it only optional
+      def initialize(pvs, pager)
+        textdomain "storage"
+        @pvs = pvs
+        @pager = pager
+      end
+
+      # table items. See CWM::Table#items
+      def items
+        @pvs.map do |pv|
+          device = pv.plain_blk_device
+          [
+            id_for_device(device), # use name as id
+            device.name,
+            device.size.to_human_string,
+            # TODO: dasd format use "X", check it
+            device.exists_in_probed? ? "" : _(BlkDevicesTable::FORMAT_FLAG),
```

Comment:
Yes, there is a difference. Just imagine a partition that was already there used for another purpose (or not used at all) and now is used as PV. Then (ignore the possible nils):
```ruby
the_partition.exists_in_probed? #=> true, so we consider the PV is not new
the_partition.lvm_pv.exists_in_probed? #=> false. That's actually what you wanted to check
```
So I will create a corresponding method in `Y2Storage::BlkDevice` to check if something is going to be "formatted" as PV.